src/client.js

```diff
@@ -2,14 +2,17 @@
 /* global __DEVELOPMENT__ */
 import 'babel-polyfill'
 import 'normalize.css'
+import { BrowserRouter, Route } from 'react-router-dom'
+import { Provider } from 'react-redux'
 import App from './app'
+import config from '../config'
+import configureStore from './store/configure-store'
 import Loadable from 'react-loadable'
 import React from 'react'
 import ReactDOM from 'react-dom'
 import ReactGA from 'react-ga'
-import configureStore from './store/configure-store'
-import { BrowserRouter, Route } from 'react-router-dom'
-import { Provider } from 'react-redux'
+
+const releaseBranch = config.releaseBranch
```

Comment:
For client side, we might need to create `src/global-env.js`[(example)](https://github.com/twreporter/membership-fe/blob/master/src/global-env.js), since `config.js` contains cookie secret which is supposed not to be exposed on the client side.

Reply:
Fixed